buffer/src/test/java/io/netty/buffer/PoolArenaTest.java

```diff
@@ -46,6 +48,7 @@ public void testNormalizeAlignedCapacity() throws Exception {
 
     @Test
     public void testDirectArenaOffsetCacheLine() throws Exception {
+        assumeTrue(PlatformDependent.hasUnsafe());
```

Comment:
@normanmaurer I added this because I noticed that it fails when running the tests with `-Dio.netty.noUnsafe=true` (I think it's actually N/A in that case). Should we include test runs with Unsafe disabled in the CI?

Reply:
@njhill yes will add it.